### ISSUE TITLE
Add more characters to game character

### ISSUE THREAD

**Issue by Oniioppai** [Mantaro/MantaroBot#184]
It would be cool if mantaro's command (->game character) would have many more characters. Me and my friends have allready remembered all the character's and its becoming really boring to see same characters over again. If you can do something about that we would be very happy for that.

**Comment by Oniioppai** [comment]
And there is a issue in (->game character stephanie from No game no life) it isnt working it allways says error.

**Comment by howdoicode5058** [comment]
its been known in the hub already :p

**Comment by MrLar** [comment]
Since you haven't received an actual answer yet (apologies for that), characters are something we don't "just" add. We encourage people to suggest a bunch **at once** in either or [support guild](https://support.mantaro.site), or my DMs (MrLar 🌸
#0611, you'll have to share a server with me I won't accept FRs). However like mentioned this should happen **at once** as in _please_ don't spam neither our suggestion channel nor my DMs (you'll be blocked).

tl;dr If you suggest a bunch and others also do we will keep expanding it every time we get enough to make a difference

**Comment by MrLar** [comment]
On further note, since this is an old issue and more so an overall thing than an actual suggestion, I'm closing this issue. Feel free to follow the steps above to suggest characters
